server/validators/participant.ts

```diff
@@ -97,13 +100,19 @@ export const ParticipantSchema = yup
   .object()
   .noUnknown('Unknown field in form')
   .shape({
+    program: yup.string().oneOf(programs, 'Program should be HCA or MHAW'),
+
     // Eligibility
     eligibility: yup
       .boolean()
       .typeError(errorMessage)
       .required(errorMessage)
       .test('is-true', errorMessage, (v) => v === true),
 
+    educationalRequirements: yup
+      .string()
+      .oneOf(yesOrNo, 'Educational requirements should be Yes, No'),
```

Comment:
education question has a 3rd option `I don't know`, it's set to `do not know` on my end